tests/Unit/Parser/PHP/Strategy/AnnotationStrategyTest.php

```diff
@@ -84,7 +97,7 @@ public function eirmod() {}
 
         self::assertSame(
             [
-                'My\Namespace\Lorem',
+                ...($this->phpStanPhpDocParserV2 ? [] : ['My\Namespace\Lorem']),
```

Comment:
@icanhazstring Looks like this specific one isn't supported in v2. Hope that isn't an issue.

Reply:
Hm. This needs to work still, because
```
/**
 * @AnnotationClass(property=value)
 */
```

should still work. Will investigate in the evening.

Reply:
Good point, I reverted this line for now.